SupportYourLocals.Data/CSVData.cs

```diff
@@ -5,73 +5,95 @@
 using MapControl;
 using Microsoft.VisualBasic.FileIO;
 using SupportYourLocals.ExtensionMethods;
+using System.Security.Cryptography;
 
 namespace SupportYourLocals.Data
 {
-    public class CSVData
+    public class CSVData : IUserStorage
     {
-        private static string filePath = @"./Data.csv";
-        private static int personsID = 1000;
+        private static string filePath = @"./UserData.csv";
 
-        public static void SaveData(String product, Location position)
+        readonly List<UserData> listUserData = new List<UserData>();
+
+        public CSVData ()
+        {
+            listUserData = LoadData();
+        }
+
+        private List<UserData> LoadData()
         {
-            var csv = new StringBuilder();
+            StringBuilder csv = new StringBuilder();
+
+            if (!File.Exists(filePath))
+            {
+                return new List<UserData>();                
+            }
+            else
```

Comment:
This is a quick return condition, which will terminate if the file doesn't exist, so the rest of the code doesn't need to be in an else block.